src/main/scala/com/lightbend/rp/sbtreactiveapp/App.scala

```diff
@@ -105,19 +110,41 @@ sealed trait LagomApp extends App {
       enablePlayHttpBinding := true,
       enableAkkaClusterBootstrap := None,
 
+      akkaClusterBootstrapEnabled :=
+        magic.Lagom.hasCluster(libraryDependencies.value.toVector),
+
       ivyConfigurations += apiTools,
 
       managedClasspath in apiTools :=
         Classpaths.managedJars(apiTools, (classpathTypes in apiTools).value, update.value),
 
       libraryDependencies ++= magic.Lagom.component("api-tools").toVector.map(_ % apiTools),
 
-      endpoints := endpoints.value ++ magic.Lagom.endpoints(
-        ((managedClasspath in apiTools).value ++ (fullClasspath in Compile).value).toVector,
-        scalaInstance.value.loader,
-        lagomIngressPorts.value,
-        lagomIngressHosts.value)
-        .getOrElse(Seq.empty))
+      // Note: Play & Lagom need their endpoints defined first (see play-http-binding)
+
+      endpoints := {
+        val ingressPorts = httpIngressPorts.value
+        val ingressHosts = httpIngressHosts.value
+        val ingressPaths = httpIngressPaths.value
+        val endpointName = name.value
+
+        val magicEndpoints =
+          magic.Lagom.endpoints(
+            ((managedClasspath in apiTools).value ++ (fullClasspath in Compile).value).toVector,
+            scalaInstance.value.loader,
+            httpIngressPorts.value,
+            httpIngressHosts.value,
+            httpIngressPaths.value)
+            .getOrElse(Seq.empty)
+
+        val autoEndpoints =
+          if (magicEndpoints.nonEmpty)
+            magicEndpoints
+          else
+            Vector(HttpEndpoint(endpointName, 0, HttpIngress(ingressPorts, ingressHosts, Vector("/"))))
```

Comment:
@longshorej - should this be `/` or `ingressPaths`?

If this should be `/`, we should remove `ingressPaths` because it's not used.

Reply:
I made this more clear - good catch.